client/landing/site.landing/coffee/core/utils.coffee

```diff
@@ -557,6 +557,14 @@ module.exports = utils = {
       else resolve response.token.id
 
 
+  submitWithDummyToken: (client) ->
+    STRIPE_TEST_TOKEN = 'tok_visa'
```

Comment:
/cc @cihangir since there is [no way to pre-fill sensitive data](https://stripe.com/docs/stripe.js#element-options) using Stripe elements, and the new way of creating token is [by passing a instance](https://stripe.com/docs/stripe.js#stripe-create-token) of a `StripeElement` to `stripe.createToken` function, they now provide test tokens to bypass this process.

This eliminates the need to make a call for creating a token.